test/python/conftest.py

```diff
@@ -52,9 +54,18 @@ def pytest_configure(config):
 # Pull all features from the base ragger conftest using the overridden configuration
 pytest_plugins = ("ragger.conftest.base_conftest", )
 
+@pytest.fixture(scope="session")
+def snapshots_path():
+    """
+    This fixture provides the default path for screenshots.
+    It is used in the ExchangeNavigationHelper.
+    """
+    # Use the current file's directory as the base path
+    return Path(__file__).parent.resolve()
+
 @pytest.fixture(scope="function")
-def exchange_navigation_helper(backend, navigator, test_name):
-    return ExchangeNavigationHelper(backend=backend, navigator=navigator, test_name=test_name)
+def exchange_navigation_helper(backend, navigator, snapshots_path, test_name):
+    return ExchangeNavigationHelper(backend=backend, navigator=navigator, snapshots_path=snapshots_path, test_name=test_name)
```

Comment:
Will this need to be added to all conftest implementing exchange tests ?

Reply:
Yes, and it is already the case in the C boilerplater swap tests.

Reply:
Let's keep it for now and maybe improve in step 2